common/src/main/java/com/revenuecat/purchases/common/errorLogUtils.kt

```diff
@@ -0,0 +1,101 @@
+package com.revenuecat.purchases.common
+
+import com.revenuecat.purchases.strings.Emojis
+
+fun googleErrorLog(level: LogLevel, message: String) {
+    val emojiList = listOf(Emojis.GOOGLE_ERROR, Emojis.DOUBLE_EXCLAMATION)
+    val emojifiedMessage = "${emojiList.joinToString("")} $message"
+    when (level) {
+        LogLevel.DEBUG -> debugLog(emojifiedMessage)
+        LogLevel.INFO -> log(emojifiedMessage)
+        LogLevel.ERROR -> errorLog(emojifiedMessage)
+    }
+}
+
+fun infoLog(level: LogLevel, message: String) {
+    val emojiList = listOf(Emojis.INFO)
+    val emojifiedMessage = "${emojiList.joinToString("")} $message"
+    when (level) {
+        LogLevel.DEBUG -> debugLog(emojifiedMessage)
+        LogLevel.INFO -> log(emojifiedMessage)
+        LogLevel.ERROR -> errorLog(emojifiedMessage)
+    }
+}
+
+fun purchaseLog(level: LogLevel, message: String) {
+    val emojiList = listOf(Emojis.PURCHASE)
+    val emojifiedMessage = "${emojiList.joinToString("")} $message"
+    when (level) {
+        LogLevel.DEBUG -> debugLog(emojifiedMessage)
+        LogLevel.INFO -> log(emojifiedMessage)
+        LogLevel.ERROR -> errorLog(emojifiedMessage)
+    }
+}
+
+fun rcSuccessLog(level: LogLevel, message: String) {
+    val emojiList = listOf(Emojis.RC_SUCCESS)
+    val emojifiedMessage = "${emojiList.joinToString("")} $message"
+    when (level) {
+        LogLevel.DEBUG -> debugLog(emojifiedMessage)
+        LogLevel.INFO -> log(emojifiedMessage)
+        LogLevel.ERROR -> errorLog(emojifiedMessage)
+    }
+}
+
+fun rcPurchaseSuccessLog(level: LogLevel, message: String) {
+    val emojiList = listOf(Emojis.RC_SUCCESS, Emojis.PURCHASE)
+    val emojifiedMessage = "${emojiList.joinToString("")} $message"
+    when (level) {
+        LogLevel.DEBUG -> debugLog(emojifiedMessage)
+        LogLevel.INFO -> log(emojifiedMessage)
```

Comment:
If you look at the implementation of `log`, we call `Log.w`, which is WARNING level of logging. In Android, the levels live in the `Log` class inside `android.util`, and there's a `VERBOSE`, `DEBUG`, `INFO`, `WARN`, `ERROR` and `ASSERT`. 

As you can see, there's an INFO level that we are not using and that could cause confusion, so I would call this `LogLevel.WARNING` or change the `log` to do `Log.i` instead of `Log.w`

Reply:
Or can we change `log` to be `Log.i` and create a new log function `warnLog` inside `logUtils` that would be for `Log.w`.